tsconfig.json

```diff
@@ -1,42 +1,35 @@
 {
+  "extends": "@canonical/typescript-config-react",
   "compilerOptions": {
     "outDir": "./public/js/dist",
     "baseUrl": ".",
     "paths": {
       "api/*": ["src/api/*"],
       "context/*": ["src/context/*"],
       "components/*": ["src/components/*"],
+      "lib/*": ["src/lib/*"],
       "pages/*": ["src/pages/*"],
       "types/*": ["src/types/*"],
       "util/*": ["src/util/*"],
       "*": ["types/*.d.ts"]
     },
-    "target": "ES2020",
```

Comment:
upstream config for `target` is set to `ES2023`, was a bit worried about browser compatibility but according to this [matrix](https://compat-table.github.io/compat-table/es2016plus/) I think it is okay.